core/src/main/java/uk/gov/gchq/koryphe/impl/predicate/AreIn.java

```diff
@@ -40,13 +40,15 @@
 @Summary("Checks if a provided collection contains all the provided input values")
 public class AreIn extends KoryphePredicate<Collection<?>> {
     private Collection<?> allowedValues;
+    private boolean isNullOrEmptyAllowed;
```

Comment:
This should default to previous behaviour, which is true

Reply:
Added